ptvsd/pydevd/_pydevd_bundle/pydevd_process_net_command.py

```diff
@@ -249,11 +249,18 @@ def process_net_command(py_db, cmd_id, seq, text):
                 # command to add some breakpoint.
                 # text is file\tline. Add to breakpoints dictionary
                 suspend_policy = "NONE"
+                continue_execution = None
```

Comment:
Is there any particular reason to have this as a tri-state bool? It seems that there are no checks that distinguish `True` and `None`.